src/pages/søknad/utils.ts

```diff
@@ -1,13 +1,13 @@
 import { Søknadstema } from '~src/types/Søknad';
 
-export const getSøknadstematekst = (
+export function getSøknadstematekst<AlderTekst extends string, UføreTekst extends string>(
```

Comment:
Det var nødvendig med en strengere type her for å kunne bruke `formatMessage` på returverdien.

Reply:
Ja, sett. Vi kunne vurdert å kalle metoden noe annet nå ettersom den egentlig brukes til mer enn å hente tekst basert på søknadstype. Men ikke så veldig farlig.

Hvordan vi skal skille på bruken av function vs const er for så vidt noe vi kunne bestemt oss for oxo:)